urllib3/poolmanager.py

```diff
@@ -110,8 +116,14 @@ class PoolManager(RequestMethods):
 
     proxy = None
 
-    def __init__(self, num_pools=10, headers=None, **connection_pool_kw):
+    def __init__(self, num_pools=10, headers=None, transport_security_manager=None,
+                 **connection_pool_kw):
         RequestMethods.__init__(self, headers)
+
+        if transport_security_manager is None:
+            transport_security_manager = TransportSecurityManager()
```

Comment:
Having to pass a no-op class seems like a somewhat painful/unpythonic solution, I feel like specifically setting a variable to `None` would suggest we wanted the feature disabled. The other uses of transport_security_manager do a check to verify it's not `None`. Could we possibly implement "on by default" with a different default or flag?